queenbee/_openapi.py

```diff
@@ -0,0 +1,98 @@
+"""This module generates the OpenAPI documentation for Queenbee workflow
+based on Pydantic implementation.
+"""
+
+from pydantic.schema import schema
+from typing import Dict
+
+from .schema.workflow import Workflow
+
+_base_open_api = {
+    "openapi": "3.0.2",
+    "servers": [],
+    "info": {
+        "description": "This is the documentation for Queenbee Workflow schema.",
+        "version": "1.7.0",
```

Comment:
Any way we can set this based on the library version rather than have it hard coded? I'm thinking we might be able to do so by running the "gh-pages" job as a github action triggered by a release event.

Reply:
I was lazy to do it right. It is now looking up for the version.